src/app/dynamic/enterprise/allowed-registries/allowed-registry-dialog/template.html

```diff
@@ -22,7 +22,7 @@
 
 <km-dialog-title>{{data.title}}</km-dialog-title>
 <mat-dialog-content>
-  <p *ngIf="data.mode === 'edit'"
+  <p *ngIf="data.mode === 'Edit'"
```

Comment:
Please use enum instead of `'Edit'` (string). Same comment for other similar changes in this PR.